t/pipe-stderr.t

```diff
@@ -783,7 +783,7 @@ end
             local f = io.open("$TEST_NGINX_HTML_DIR/a.lua")
             local code = f:read("*a")
             local proc = helper.run_lua_with_graceful_shutdown("$TEST_NGINX_HTML_DIR", code)
-            proc:set_timeouts(100, 100, 100, 100)
+            proc:set_timeouts(1000, 1000, 1000, 1000)
```

Comment:
It's usually not a good idea to let tests run this long. How about introducing a shorter file for reading here? `a.lua` is a bit large.

Reply:
I've changed the timeout time to 300ms and found it is good enough for testing stability.
`a.lua` will not be adjusted for this time.